ssi-bot_template.ini

```diff
@@ -11,3 +11,13 @@ model_path = models/path_to_model/
 ; will result in your bot being banned
 negative_keywords=
 positive_keywords=
+
+; REQUIRED, the name of the subreddit to post to
+subreddit = test
+
+; REQUIRED, the id of the flair to use for posts
```

Comment:
Remove 'required' because it's not.
Also let's put the subsimgpt2 'post by bot' flair id in as a comment, or default, so the user doesn't have to find it themselves.

Reply:
1. Well, IMHO, all of these (apart from the flair id) should be filled out, and if not then it should warn the user to fill them out properly, which is why I used `logging.warning` as opposed to `logging.info`.
2. Yeah, thats probably a good idea.

Reply:
Yep sorry, my fault on that. I agree the others are required and only flair id is not required!

Reply:
Done.

Reply:
Whoops, resolved wrong one

Reply:
Done.